api/query/query.go

```diff
@@ -145,7 +167,7 @@ func (qh queryHandler) loadSummary(testRun shared.TestRun) ([]byte, error) {
 }
 
 func getRedisKey(testRun shared.TestRun) string {
-	return "RESULTS_SUMMARY-" + strconv.FormatInt(testRun.ID, 10)
+	return "RESULTS_SUMMARY-v2-" + strconv.FormatInt(testRun.ID, 10)
```

Comment:
We definitely will need to flush the cache prior to landing this change. I'm not sure of the protocol we have set up currently for cache data expiration, but this will certainly invalidate all of these old keys.